### DISCUSSION TITLE
Create a column in Access Table with "long text"

### DISCUSSION THREAD

**Discussion by AndreasWaning** [linq2db/linq2db#4004] [Q&A] [answered]
If I create a Table in MS Access with the following Poco Class, the Text columnField is from Type "short text 255 chars"

How can I set it to "long text"  ?

Thanks Andreas

	[Table("LiquidationTextGerichtlich")]
	public class LiquidationTextGerichtlich
	{
		[Column("Nummer"    , IsPrimaryKey = true)] public int    Nummer     { get; set; } // int
		[Column("Text"                           )] public string Text       { get; set; } // nvarchar(max)
		[Column("Bermerkung"                     )] public string Bermerkung { get; set; } // nvarchar(max)
	}

**Comment by MaceWindu** [comment, marked as answer]
you can specify type using `ColumnAttribute.DbType` property

```cs
[Column("Text", DbType = "long text")] 
```

**Comment by AndreasWaning** [comment]
Thanks Mark.

Error : Can not covert "long Text" to DbType.

How can I create a nvarchar(max) Field ?

Tried this:

[Column("Text", DbType = DataType.NVarchar,length=int.MaxValue)] 

Than he said "Field to long.

Have you an idea ?

**Reply by MaceWindu** [reply]
Have you seen my suggestion? DbType is a string property, which accepts database type name. Not sure why you try to use it with DataType enum

**Reply by AndreasWaning** [reply]
Sorry, my fault. Thank you
